### DISCUSSION TITLE
Error response from daemon: manifest for public.ecr.aws/supabase/gotrue:2.139.2-rc.7 not found: manifest unknown: Requested image not found

### DISCUSSION THREAD

**Discussion by developer-espark** [supabase/supabase#21449] [Questions]
Dear Supabase Support Team,

I am currently experiencing an issue related to supabase CLI. In this issue Requested image not found error occurred. 

![image](https://github.com/supabase/supabase/assets/52691329/96cf062d-19a8-439f-8b12-92fb9b90cb40)

Also having trouble connecting with the following issue: network is unreachable

![image](https://github.com/supabase/supabase/assets/52691329/f6b7e6ec-b2e5-40ef-bf05-b919de031397)

**Comment by GaryAustin1** [comment]
On your first issue:  https://github.com/supabase/cli/issues/1938

For your 2nd issue possibly this:
![image](https://github.com/supabase/supabase/assets/54564956/7146dd55-811b-4279-88c8-9a330d3ecdfa)
from: https://github.com/orgs/supabase/discussions/17817

**Reply by webdeveloper20213** [reply]
@GaryAustin1 Thanks for reply,

1. first issue was resolve 
2. for secound issue as of now over network is ipv4 is there any solution to do command line interface(CLI) using ipv4 or is there any other way?

**Reply by GaryAustin1** [reply]
To use ipv4 you need to follow the instructions on using supavisor.  The info I posted and the link has the most info on troubleshooting.

**Reply by webdeveloper20213** [reply]
@GaryAustin1 Thanks for reply,

if we enabled Dedicated IPv4 address in supabase, the following problem emerged (**dial error (timeout: dial tcp 54.90.141.8:5432: i/o timeout)**)

see bellow image

![image](https://github.com/supabase/supabase/assets/133633860/6829a642-d8bf-4b0f-aead-f000c7007a5d)
